### DISCUSSION TITLE
Generate PDF with Page Outlines

### DISCUSSION THREAD

**Discussion by seanmrnda** [HollowMan6/mdbook-pdf#2] [Ideas]
Can we generate the pdf and keep the outlines?

**Comment by HollowMan6** [comment]
I think I've explained clearly in #1 , if you have to address that further, why bother start a new issue and a discussion for that instead of just continue commenting under #1?

Please read the links in here: https://github.com/HollowMan6/mdbook-pdf/issues/1#issuecomment-1165560732 , especially the Chromium issue:

> This should be realized by Chromium, and [an issue has already been filed for this here](https://bugs.chromium.org/p/chromium/issues/detail?id=781797).

And I had also mentioned this in the README [common Issues part 3](https://github.com/HollowMan6/mdbook-pdf#common-issues) before you post #1. If you have really strong interest, you can help Chromium realize that.

I'm closing this issue and the discussion, if you have more to share, you can comment under #1, Cheers!
